website/blog/detox-20-is-out.md

```diff
@@ -0,0 +1,260 @@
+
+### Reset lock file
+
+This release adds a small CLI tool, [`detox reset-lock-file`][reset-lock-file], to help users with one specific use scenario.
+
+Imagine you want to run tests for multiple Detox configurations at once, e.g.:
+
+```bash
+detox test -c iphoneSE2020.release e2e/ui.test.js
+detox test -c iphone14ProMax.release e2e/ui.test.js
+```
+
+The problem is that Detox uses a file-locking mechanism to avoid situations when parallel test workers would take control of the same device. The `detox test` command, upon start, erases that file contents, creating a race condition risk.
+
+To eliminate that risk, use a combination of `detox reset-lock-file` and `--keepLockFile` like this:
+
+```bash
+detox reset-lock-file & \
+detox test --keepLockFile -c iphoneSE2020.release e2e/ui.test.js & \
+detox test --keepLockFile -c iphone14ProMax.release e2e/ui.test.js & \
+wait
+```
+
+## Deprecations
+
+Detox 20 executes many pending deprecations, so make sure to check out our [Migration Guide] before upgrading:
+
+* JS: minimum supported Node.js version is `14.x`;
+* JS: minimum supported Jest version is `27.2.5`;
+* JS: Mocha test runner is no longer supported;
+* JS: discontinued old adapters for Jest (`jest-jasmine`, first generation of `jest-circus` adapter);
+* JS: discontinued `{ permanent: true }` option in `device.appLaunchArgs.*` methods ([#3360](https://github.com/wix/Detox/pull/3360));
+* Config: discontinued kebab-case properties: `test-runner`, `runner-config` ([#3371](https://github.com/wix/Detox/pull/3371))
+* Config: discontinued `skipLegacyWorkersInjections` property ([(#3286)](https://github.com/wix/Detox/pull/3286))
+* Config: deprecated `specs` and `runnerConfig` properties
+* Config: changed [semantics][`Config file > Test runner`] of `testRunner` property
+* Config: dropped support for all-in-one configurations ([#3386](https://github.com/wix/Detox/pull/3386));
+* Android: remove deprecated native IdlePolicyConfig ([#3332](https://github.com/wix/Detox/pull/3332/files))
+* iOS: discontinued `ios.none` device type ([#3361](https://github.com/wix/Detox/pull/3361))
+
+## Afterword
+
+
+Over the last year and a half, we have established a centralized configuration system for more than 50 projects using Detox at Wix. While it never seemed to be a cakewalk, the entire experience of troubleshooting over a hundred issues across the organization did not leave us unchanged.
+
+We see numerous things to improve in Detox, but most of them boil down to the same thing – **scaling**. Surprisingly, "scaling" makes an excellent umbrella term for nearly every challenge we've been encountering lately:
+
+* _scaling up the users' count_ requires us to improve the onboarding and troubleshooting experience;
+* _scaling up the projects' count_ forces us to centralize scattered configs into flexible organization presets;
+* _scaling up the tests' count_ prompts us to optimize the codebase and incline it towards cloud and remote execution.
+
```

Comment:
Not a total expert but I thing using single-form `user`, `project`, `test` would also be right here

Reply:
Rephrased to "the number of X"